src/calls/components/Calendar.jsx

```diff
@@ -1,310 +1,50 @@
-import React, { Component } from 'react';
-import {
-    Dropdown,
-    DropdownButton,
-    ButtonToolbar,
-    Table
-} from 'react-bootstrap';
-import {
-    startOfMonth,
-    getDaysInMonth,
-    getDay,
-    format,
-    addMonths,
-    subMonths,
-    addDays,
-    setMonth,
-    setDate,
-    setYear,
-    startOfWeek,
-    lastDayOfWeek,
-    lastDayOfMonth,
-    endOfMonth,
-    subYears
-} from 'date-fns';
-import { IoIosArrowBack, IoIosArrowForward } from 'react-icons/io';
+import React from 'react';
+import Calendar from 'react-calendar';
```

Comment:
There are a few reasons why we worked on a custom calendar before. I can go into details live, but the long and short of it is that the calendar is a central component of the application, and we need to make sure that we can support any custom behavior required in the future. There are plans for this already in the pipeline. 🙂